<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFMultiConverter</class>
 <widget class="QMainWindow" name="FFMultiConverter">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>295</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>FF Multi Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Επιλογή αρχείου:</string>
          </property>
          <property name="buddy">
           <cstring>openToolButton</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fromLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="openToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Μετατροπή αρχείου:</string>
          </property>
          <property name="buddy">
           <cstring>typeComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Μετατροπή από:</string>
          </property>
          <property name="buddy">
           <cstring>fromComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="fromComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Μετατροπή σε:</string>
          </property>
          <property name="buddy">
           <cstring>toComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="toComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="folderCheckBox">
            <property name="text">
             <string>Μετατροπή όλων των
αρχείων στον φάκελο </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="recursiveCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Μετατροπή όλων των 
αρχείων αναδρομικά</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="deleteCheckBox">
            <property name="text">
             <string>Διαγραφή 
προτοτύπου</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="typeRadioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Ίδιου τύπου</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="extRadioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Ίδιας κατάληξης</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="convertPushButton">
          <property name="text">
           <string>&amp;Μετατροπή</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKati">
    <property name="title">
     <string>Αρχείο</string>
    </property>
    <addaction name="convertAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Βοήθεια</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuKati"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="text">
    <string>Έξοδος</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="convertAction">
   <property name="text">
    <string>Μετατροπή</string>
   </property>
   <property name="toolTip">
    <string>Μετατροπή</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>Περί</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+?</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>fromLineEdit</tabstop>
  <tabstop>openToolButton</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>fromComboBox</tabstop>
  <tabstop>toComboBox</tabstop>
  <tabstop>folderCheckBox</tabstop>
  <tabstop>recursiveCheckBox</tabstop>
  <tabstop>deleteCheckBox</tabstop>
  <tabstop>typeRadioButton</tabstop>
  <tabstop>extRadioButton</tabstop>
  <tabstop>convertPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
